### PR TITLE
Add tracing to PXF with Spring Sleuth

### PR DESCRIPTION
**Pull request by frankgh**
Add X-B3-TraceId and X-B3-SpanId to client and log the traces on the server logs

### CODE REVIEW

File: server/pxf-service/src/main/resources/application.properties [hunk 1]
```diff
@@ -8,13 +8,18 @@ management.endpoints.web.exposure.include=health,info,shutdown,metrics,prometheu
...
+
 # common tags applied to all metrics
 management.metrics.tags.application=pxf-service
 
 spring.profiles.active=default
 
 server.port=${pxf.port:5888}
 
+# sleuth
+spring.sleuth.trace-id128=true
```
Comment (bradfordb-vmware) [inline server/pxf-service/src/main/resources/application.properties:21]: Do we want to leave a comment here explaining why we decided to use 128-bit trade IDs?

**Review by bradfordb-vmware — APPROVED** [review]

File: server/gradle.properties [hunk 2]
```diff
@@ -27,6 +27,7 @@ pxfProtocolVersion=v15
 parquetVersion=1.11.1
 awsJavaSdk=1.11.801
 spring_boot_version=2.4.3
+springCloudVersion=2020.0.1
```
Comment (ashuka24) [inline server/gradle.properties:30]: we seem to be inconsistent with using underscores, camelCase vs . between words. Is there a method to the madness that I'm missing or would it be possible to make these all consistent?

File: server/pxf-service/src/main/resources/application.properties [hunk 3]
```diff
@@ -8,13 +8,18 @@ management.endpoints.web.exposure.include=health,info,shutdown,metrics,prometheu
 management.endpoint.shutdown.enabled=true
 management.endpoint.health.probes.enabled=true
 
+spring.application.name=pxf-service
```
Comment (ashuka24) [inline server/pxf-service/src/main/resources/application.properties:11]: why is this needed now and not needed before?

**Comment by frankgh** [comment]
Closing for the moment until we concretely define use cases, and ways for users to consume tracing information.
